### PR TITLE
Full AggLayer integration how to

### PR DESCRIPTION
**Pull request by kmurphypolygon**
- We were missing so many details.
- I have added the full steps.
- I have also amended the RPC render a little to look nicer.

### CODE REVIEW

File: docs/cdk/how-to/agglayer/integrate.md
Comment (EmpieichO) [inline docs/cdk/how-to/agglayer/integrate.md:1]: It is not clear what data this is. Is it the user's network data or the CDK data?

**Review by EmpieichO — APPROVED** [review]

File: docs/cdk/how-to/agglayer/integrate.md
Comment (kmurphypolygon) [reply docs/cdk/how-to/agglayer/integrate.md:1]: Can you be more specific please @EmpieichO I don't know what you're referring to.

File: docs/cdk/how-to/agglayer/integrate.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+## Step 3: External CDK chain configuration
+
+The last step is to update your node configuration to map to the Polygon Agglayer.
+
+Open the `config.toml` file and include the following:
+
+```sh
+[Aggregator]
+SettlementBackend = "agglayer"
+AggLayerURL = "https://agglayer-test.polygon.technology"
```
Comment (Sharonbc01) [inline docs/cdk/how-to/agglayer/integrate.md]: @kmurphypolygon Please do not provide the actual Url as we only plan to share directly with each chain. Can you put in a generic entry like this http://agglayer-001.{{ base_dn }}:{{ zkevm_agglayer_server_port }}

File: docs/cdk/how-to/agglayer/integrate.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+AggLayerTxTimeout = "300s"
+SenderAddress = "0x0325686a18aA829B9FaaAD70f22ea0830aA6076F"
```
Comment (Sharonbc01) [inline docs/cdk/how-to/agglayer/integrate.md]: @kmurphypolygon  Please do not provide the wallet address as we only plan to share directly with each chain. Can you put in a generic entry like "0x0000000000000000000000000000000000000000000"

File: docs/cdk/how-to/agglayer/integrate.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+Once registered, Polygon creates a permissionless node that points to the corresponding L2 full node.
+
+We then add your details to our AggLayer configuration file [`../agglayer/docker/data/agglayer.toml`](https://github.com/0xPolygon/agglayer/blob/main/docker/data/agglayer/agglayer.toml); specifically these configurations:
+
+* `[FullNodeRPCs]` points to the corresponding L2 full node.
+* `[L1]` points to the corresponding L1 chain.
+* The `[DB]` section has the managed database details.
+
+For example:
```
Comment (Sharonbc01) [inline docs/cdk/how-to/agglayer/integrate.md]: @kmurphypolygon  can we remove this example config file here as this is internal to Polygon and would prefer not to expose. Thanks

File: docs/cdk/how-to/agglayer/integrate.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+
+    - `rollupID`
+    - `rollupAddress`
+
+3. We also give you a pre-configured `genesis.json` file to use.
+
+4. You will need the Polygon bridge configuration details.
+
+    ```sh
+    [NetworkConfig]
```
Comment (Sharonbc01) [inline docs/cdk/how-to/agglayer/integrate.md]: Hi @kmurphypolygon  can you check https://github.com/0xPolygonHermez/zkevm-bridge-service/blob/develop/config/config.debug.toml#L50     I think a couple of lines might be missing.

File: docs/cdk/how-to/agglayer/integrate.md (hunk 1 above)
Comment (kmurphypolygon) [reply docs/cdk/how-to/agglayer/integrate.md]: done

File: docs/cdk/how-to/agglayer/integrate.md (hunk 2 above)
Comment (kmurphypolygon) [reply docs/cdk/how-to/agglayer/integrate.md]: done

File: docs/cdk/how-to/agglayer/integrate.md (hunk 3 above)
Comment (kmurphypolygon) [reply docs/cdk/how-to/agglayer/integrate.md]: done

File: docs/cdk/how-to/agglayer/integrate.md (hunk 4 above)
Comment (kmurphypolygon) [reply docs/cdk/how-to/agglayer/integrate.md]: done added, i skipped some that looked non-global, perhaps they're all global??

**Comment by Sharonbc01** [comment]
Thanks @kmurphypolygon the changes look good. I have an engineer reviewing and updating the source doc https://www.notion.so/polygontechnology/CDK-setup-uLxLy-and-AggLayer-afc9bcc788db41319dac4cc2c6213d3e  so I would like to do another review later this week / early next before we merge.

**Comment by kmurphypolygon** [comment]
> Thanks @kmurphypolygon the changes look good. I have an engineer reviewing and updating the source doc https://www.notion.so/polygontechnology/CDK-setup-uLxLy-and-AggLayer-afc9bcc788db41319dac4cc2c6213d3e so I would like to do another review later this week / early next before we merge.

OK, in that case I'm putting this PR into DRAFT.

File: docs/cdk/how-to/agglayer/integrate.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+* `[FullNodeRPCs]` points to the corresponding L2 full node.
+* `[L1]` points to the corresponding L1 chain.
+* The `[DB]` section has the managed database details.
+
+## Step 3: External CDK chain configuration
+
+The last step is to update your node configuration to map to the Polygon Agglayer.
+
+Open the `config.toml` file and include the following:
```
Comment (Sharonbc01) [inline docs/cdk/how-to/agglayer/integrate.md:27]: Please add to this sentence. Polygon will supply exact URL and wallet address directly to the IP Partner

**Comment by Sharonbc01** [comment]
Hi @kmurphypolygon here is the source doc in Notion for a review against by the docs team thanks. https://www.notion.so/polygontechnology/CDK-setup-uLxLy-and-AggLayer-afc9bcc788db41319dac4cc2c6213d3e

File: docs/cdk/how-to/agglayer/integrate.md (hunk 5 above)
Comment (kmurphypolygon) [reply docs/cdk/how-to/agglayer/integrate.md:27]: I assumed you meant `SenderAddress` here.

**Comment by kmurphypolygon** [comment]
> Hi @kmurphypolygon here is the source doc in Notion for a review against by the docs team thanks. https://www.notion.so/polygontechnology/CDK-setup-uLxLy-and-AggLayer-afc9bcc788db41319dac4cc2c6213d3e

Think I nearly had everything apart from the last chunk of params which I added. A lot of these instructions are Polygon side only.

File: docs/cdk/how-to/agglayer/integrate.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+AggLayerTxTimeout = "30s"
+SequencerPrivateKey = {Path = "/etc/zkevm/sequencer.key", Password = "{{ zkevm_keystore_password }}"}
+SenderAddress = ""
+```
+
+- `SettlementBackend` defines how a final zk-proof is settled. All chains connected to the AggLayer use the `agglayer` value.
+- `AggLayerTxTimeout` is the interval a transaction is mined on the AggLayer.
+- `AggLayerURL` is the url of the agglayer service. Supplied by Polygon.
+- `SequencerPrivateKey` is the private key of the trusted sequencer. Only a CDK trusted sequencer can interact with the AggLayer.
+- `SenderAddress` is the address specified by the aggregator when it creates a proof. Supplied by Polygon. Supplied by Polygon.
```
Comment (kmurphypolygon) [inline docs/cdk/how-to/agglayer/integrate.md]:
```suggestion
- `SenderAddress` is the address specified by the aggregator when it creates a proof. Supplied by Polygon.
```

File: docs/cdk/how-to/agglayer/integrate.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+This document shows you how to integrate and configure the AggLayer into your stack.
+
+!!! important
+    - Polygon manages the AggLayer in production at the current time.
+    - In the future, the AggLayer will be decentralized.
+
+!!! warning
+    - The AggLayer is in development and subject to architectural changes.
+    - The code is still being audited.
+
```
Comment (kmurphypolygon) [inline docs/cdk/how-to/agglayer/integrate.md:12]: @Sharonbc01 Are we missing a step here:

**Prerequisites** 

You should have a CDK stack set up that has integrated the bridge service. See [instructions]..

**Review by EmpieichO — APPROVED** [review]

**Pull request merged**